shell/browser/main.js

```diff
@@ -217,7 +205,6 @@ class Browser {
       const selectedId = win.tabs.selected ? win.tabs.selected.id : -1
       Object.assign(tabInfo, {
         active: tabInfo.id === selectedId,
-        windowId: win.id,

```

Comment:
I think this might have been left out on accident. This is still needed for the extension tab info https://developer.chrome.com/extensions/tabs#type-Tab

Reply:
The equivalent value is set there using `getParentWindowOfTab`:
https://github.com/samuelmaddock/electron-browser-shell/blob/87b77f4eeffe0e350733825e12784b5b21aeed8a/shell/browser/extensions.js#L190